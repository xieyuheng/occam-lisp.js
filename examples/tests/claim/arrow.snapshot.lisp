(-> int? int? int?)
(-> int? (-> int? int?))

(assert-equal
  (-> int? int? int?)
  (-> int? (-> int? int?)))

((-> int? int? int?) iadd)

(assert-equal
  (((-> int? int? int?) iadd) 1 1)
  2)
