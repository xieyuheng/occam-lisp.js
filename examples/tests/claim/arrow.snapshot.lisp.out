(-> int? int? int?)
(-> int? (-> int? int?))
(the (-> int? int? int?) iadd)
